stylesheets/_component.rules.scss

```diff
@@ -298,3 +298,338 @@ td.table__form .rules + .form__group:last-of-type {
+        @include respond-min($screen-tablet) {
+            padding: 0;
+        }
+    }
+
+    .rule-block__heading {
+        padding: 0;
+    }
+
+    .rule-block--if,
+    .rule-block--else-if {
+        background-color: lighten($rule-color-if, 40);
+        border-color: $rule-color-if;
+        border-top-left-radius: 15px;
+        border-top-right-radius: 15px;
+
+        .rule-block__header {
+            background-color: $rule-color-if;
+            border-top-left-radius: 14px;
+            border-top-right-radius: 14px;
+
+            @include respond-min($screen-tablet) {
+                border-top-right-radius: 0;
+            }
+        }
+
+        .remove-control {
+            color: color(white);
+
+            @include respond-min($screen-tablet) {
+                color: color(text);
+            }
+        }
+    }
+
+    .rule-block--else-if {
+        &::after {
+            border-top: 4px solid color(gray, lightest);
+            content: '';
+            display: block;
+            height: 4px;
+            left: -25px;
+            position: absolute;
+            top: 35px;
+            width: 24px;
+        }
+
+        .rule-block__header {
+            border-top: 0;
+        }
+    }
+
+    .rule-block--and {
+        background-color: lighten($rule-color-and, 20);
+        border-color: darken($rule-color-and, 10);
+        margin-top: 0;
+
+        .rule-block__header {
+            background-color: $rule-color-and;
+            color: color(black);
+        }
+
+        .remove__control {
+            color: color(text);
+        }
+    }
+
+    .rule-block--or {
+        background-color: lighten($rule-color-or, 25);
+        border-color: $rule-color-or;
+
+        .rule-block__header {
+            background-color: $rule-color-or;
+        }
+    }
+
+    .rule-block--else {
+        background-color: lighten($rule-color-if, 30);
+        border-color: $rule-color-if;
+
+        .rule-block__header {
+            background-color: $rule-color-if;
+        }
+
+        + .rule-block--return {
+            margin-left: 30px;
+            width: calc(100% - 30px);
+
+            &::before {
+                display: none;
+            }
+        }
+    }
+
+    .rule-block--return {
+        background-color: lighten($rule-color-return, 30);
+        border-color: $rule-color-return;
+        border-bottom-left-radius: 15px;
+        border-bottom-right-radius: 15px;
+
+        .rule-block__header {
+            background-color: $rule-color-return;
+
+            @include respond-min($screen-tablet) {
+                border-bottom-left-radius: 14px;
+            }
+        }
+    }
+
+    .rule-block--else-return {
+        background-color: color(gray, lightest);
+        border-color: color(gray, dark);
+        border-radius: 15px;
+
+        .rule-block__header {
+            background-color: color(gray, dark);
+            border-top-left-radius: 14px;
+            border-top-right-radius: 15px;
```

Comment:
Looks like this should be 14px. Not 15px.

Reply:
Cheers, fixed!